sdk/resourcemanager/Azure.ResourceManager/tests/Samples/Sample2_ManagingResourceGroups.cs

```diff
@@ -76,9 +76,8 @@ public async Task ListAllResourceGroups()
             Subscription subscription = armClient.DefaultSubscription;
             // Now we get a ResourceGroup container for that subscription
             ResourceGroupContainer rgContainer = subscription.GetResourceGroups();
-            // With ListAsync(), we can get a list of the resources in the container
-            AsyncPageable<ResourceGroup> response = rgContainer.GetAllAsync();
-            await foreach (ResourceGroup rg in response)
+            // With GetAllAsync(), we can get a list of the resources in the container
+            await foreach (ResourceGroup rg in rgContainer.GetAllAsync())
             {
                 Console.WriteLine(rg.Data.Name);
             }
```

Comment:
is it possible to add a CreateVM Sample to this samples folder?

Reply:
I think what we want to do is link to that at the bottom in the more sample code maybe?  That example is being created in the Azure.ResourceManager.Compute project and I want to try to avoid a duplicate example that might deviate.